airlock/templates/file_browser/csv.html

```diff
@@ -27,19 +28,11 @@
         <thead>
           <tr>
             {% for header in headers %}
-              <th>{{ header }}
-                <span class="ml-auto">
-                  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" class="h4 w-4 datatable-icon--no-sort" role="presentation">
-                    <path fill-rule="evenodd" d="M2.24 6.8a.75.75 0 001.06-.04l1.95-2.1v8.59a.75.75 0 001.5 0V4.66l1.95 2.1a.75.75 0 101.1-1.02l-3.25-3.5a.75.75 0 00-1.1 0L2.2 5.74a.75.75 0 00.04 1.06zm8 6.4a.75.75 0 00-.04 1.06l3.25 3.5a.75.75 0 001.1 0l3.25-3.5a.75.75 0 10-1.1-1.02l-1.95 2.1V6.75a.75.75 0 00-1.5 0v8.59l-1.95-2.1a.75.75 0 00-1.06-.04z" clip-rule="evenodd" />
-                  </svg>
-                  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" class="h4 w-4 datatable-icon--ascending" role="presentation">
-                    <path fill-rule="evenodd" d="M10 17a.75.75 0 01-.75-.75V5.612L5.29 9.77a.75.75 0 01-1.08-1.04l5.25-5.5a.75.75 0 011.08 0l5.25 5.5a.75.75 0 11-1.08 1.04l-3.96-4.158V16.25A.75.75 0 0110 17z" clip-rule="evenodd" />
-                  </svg>
-                  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" class="h4 w-4 datatable-icon--descending" role="presentation">
-                    <path fill-rule="evenodd" d="M10 3a.75.75 0 01.75.75v10.638l3.96-4.158a.75.75 0 111.08 1.04l-5.25 5.5a.75.75 0 01-1.08 0l-5.25-5.5a.75.75 0 111.08-1.04l3.96 4.158V3.75A.75.75 0 0110 3z" clip-rule="evenodd" />
-                  </svg>
-                </span>
-
+              <th>
+                <div class="flex flex-row gap-2">
+                  {{ header }}
+                  {% datatable_sort_icon %}
```

Comment:
Is there a reason for putting this in a template tag and not just as an included html snippet?

Reply:
Hmm, it is used multiple times isn't it? Maybe I've answered my own question

Reply:
Yeah, I didn't fancy copy pasting it.

I actually think the js should inject it, ideally, so that if the js fails, we're not left with these left over non-active icons.